scripts/push-docker-pr.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+# See LICENSE.txt for license information.
+
+set -e
```

Comment:
I think for consistency will be good to use one format (short or long) for setting thes